tck/running_the_tck.asciidoc

```diff
@@ -14,38 +15,91 @@
 
 = Running the Microprofile LRA TCK
 
-The TCK provides a JAX-RS participant resource together with a resource for invoking the test sutie.
+The TCK uses `JUnit 4` and `Arquillian`.
+
+== Configuration
+
+The LRA TCK suite can be parametrized by following properties, handled in the suite with MicroProfile Config
+
+`lra.tck.base.url`::
+  The URL where the TCK suite deployment is exposed at. The TCK suite will construct path based on this URL.
+  The default base URL is `http://localhost:8180`.
+`lra.tck.timeout.factor`::
+  Timeout factor of default value `1.0`. When set bigger than `1.0` then timeout value will be bigger and waiting time are longer.
+  When set-up lower then the timeouts will be shorter.
+`lra.tck.coordinator.waiting`::
+  Time that the tests waits for the LRA recovery starts-up.
+`lra.http.host`, `lra.http.port`::
+  Hostname and port where coordinator is available at. This is used for checking if tests can be started as endpoint is ready.
+`lra.http.recovery.host`, `lra.http.recovery.port`, `lra.http.recovery.path`::
+  Hostname, port and path for the recovery endpoint that will be contacted in tests checking recovery capabilities.
 
-== Dependencies
+== Prerequisites for the MicroProfile LRA TCK implementation
 
-To enable the tests in your implementation of this specification you need to add the
-following dependency to your build:
+* `pom.xml` dependencies are set-up
+* MicroProfile Config is providedby runtime for the LRA TCK suite could be run
```

Comment:
This doesn't read very well. Instead I would just say:

> MicroProfile Config

Or:

> the LRA TCK suite requires MicroProfile Config

Reply:
thanks!